templates/desktop.html

```diff
@@ -0,0 +1,163 @@
+{% extends '_layout.html' %} {% block title%} Ubuntu Desktop certified hardware
+{% endblock %} {% block content %}
+
+<nav role="navigation" class="nav-secondary clearfix">
+  <ul class="breadcrumb">
+    <li class="active">
+      <a href="/desktop/">
+        Ubuntu Desktop
+      </a>
+    </li>
+  </ul>
+</nav>
+
+<div class="row row-hero no-border">
+  <div class="container-inner title">
+    <h1>Ubuntu Desktop certified hardware</h1>
+  </div>
+
+  <div class="main">
+    <p class="large">
+      As the world’s most popular open-source desktop operating system, Ubuntu
+      is used extensively throughout the enterprise, government, public and
+      education sectors. To meet the demand for preloaded hardware from these
+      areas, Canonical is committed to working closely with high-profile
+      hardware partners around the world. Customers can choose from hundreds of
+      PC configurations that ship preloaded with Ubuntu LTS (Long Term Support),
+      bringing all the major benefits of factory-level quality assurance
+      standards, full support, and tight integration with BIOS, component and
+      board-level certification.
+    </p>
+    <p class="large">
+      Preloaded Ubuntu LTS OEM versions offer full Canonical factory-level
+      certification, BIOS, component integration and support, where benefits
+      include full support for security patches, bug fixes and more for five
+      years from the date of each LTS release.
+    </p>
+    <p class="large">
+      The certification chart below highlights OEM partner PCs available
+      preloaded with Ubuntu. Certified OEM images are also available for Ubuntu
+      Advantage customers.
+    </p>
+
+    <div class="twelve-col cosy-fit">
+      <div class="three-col">
+        <form action="/desktop/models/" method="get" class="body-search">
```

Comment:
can you replace links with `url_for('NAME OF THE FUNCTION')` this also makes it easier when we decide to change some path

Reply:
(same for all href etc.)

Reply:
Ah nice! Will update all the ones that already have functions. Ill put it into my TODO to update the other links as soon as their views are ready :)